packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -45,17 +51,17 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, any> {
         onMouseLeave={ this._onTooltipMouseLeave }
       >
         { children }
-        { isTooltipVisible ? (
+        { isTooltipVisible && (
           <Tooltip
             delay={ delay }
             content={ content }
             targetElement={ this._tooltipHost }
             directionalHint={ directionalHint }
-            calloutProps={ { onDismiss: this._onTooltipCallOutDismiss } }
+            calloutProps={ assign({ onDismiss: this._onTooltipCallOutDismiss }, calloutProps) }
```

Comment:
if a user overrides `onDismiss` will this not break the behavior of the tooltip host? Should we change the order?